ui/components/app/gas-timing/gas-timing.component.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Typography from '../../ui/typography/typography';
+import { TYPOGRAPHY } from '../../../helpers/constants/design-system';
+import InfoTooltip from '../../ui/info-tooltip';
+
+export default function GasTiming({ text, tooltipText, attitude }) {
+  return (
+    <Typography
+      variant={TYPOGRAPHY.H7}
+      className={classNames('gas-timing', {
+        [`gas-timing--${attitude}`]: attitude,
+      })}
+    >
+      {text}
+      {tooltipText && <InfoTooltip position="top" contentText={tooltipText} />}
+    </Typography>
+  );
+}
+
+GasTiming.propTypes = {
+  text: PropTypes.string,
```

Comment:
For props that are required, can we use the `.isRequired` prop check? https://reactjs.org/docs/typechecking-with-proptypes.html

In this case should be at least this one:
```
text: PropTypes.string.isRequired,
```